src/core/ColorPicker/ColorBuilder.tsx

```diff
@@ -69,29 +69,28 @@ export const ColorBuilder = React.forwardRef(
     const [colorDotActive, setColorDotActive] = React.useState(false);
     const hueColorString = hueSliderColor.toHexString();
     const colorSquareStyle = getWindow()?.CSS?.supports?.(
-      `--hue: ${hueColorString}`,
+      `--iui-color-field-hue: ${hueColorString}`,
     )
       ? {
-          '--hue': hueColorString,
-          '--selected-color': dotColorString,
+          '--iui-color-field-hue': hueColorString,
+          '--iui-color-picker-selected-color': dotColorString,
         }
       : { backgroundColor: hueColorString };
 
     const opacitySliderStyle = getWindow()?.CSS?.supports?.(
-      `--selected-color: ${hueColorString}`,
+      `--iui-color-picker-selected-color: ${hueColorString}`,
     )
-      ? { '--selected-color': hueColorString }
+      ? { '--iui-color-picker-selected-color': hueColorString }
       : { backgroundColor: hueColorString };
 
     const squareTop = 100 - hsvColor.v;
     const squareLeft = hsvColor.s;
 
     const colorDotStyle = getWindow()?.CSS?.supports?.(
-      `--top: ${squareTop.toString()}%`,
+      `--iui-color-dot-inset: 0`,
```

Comment:
Why is this `--iui-color-dot-inset: 0` instead of `--iui-color-dot-inset: ${squareTop.toString()}`? Because it's just a test to see if `inset` is supported?

Reply:
Yes, it's just a test